test/unit/test_list.c

```diff
@@ -509,6 +509,17 @@ bool test_rz_list_find_ptr(void) {
 	mu_end;
 }
 
+bool test_rz_list_sorted_uniq() {
+	const char *test_strings[] = { "cccc", "cccc", "cccc", "bbbb", "aaaa", "aaaa" };
+	RzList *list = rz_list_new_from_array((const void **)test_strings, RZ_ARRAY_SIZE(test_strings));
+	rz_list_sorted_uniq(list, (RzListComparator)strcmp, NULL);
+	mu_assert_eq(rz_list_length(list), 3, "unique strings");
+	mu_assert_streq(rz_list_first(list), "cccc", "first");
+	mu_assert_streq(rz_list_get_n(list, 1), "bbbb", "second");
+	mu_assert_streq(rz_list_last(list), "aaaa", "third");
+	mu_end;
```

Comment:
Please free the list though. Working with LSAN enabled is annoying otherwise.

Reply:
Oh sorry I forgot b4f09bb9441fafa60e11c31460f104f007f251c6